- workaround for https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=280210
- code uses the c11 feature stdatomic.h while it uses --std=c99

--- meson.build.orig	2024-11-15 20:43:07 UTC
+++ meson.build
@@ -5,7 +5,7 @@ project('libRIST', 'c',
 
 project('libRIST', 'c',
 	version: '0.2.11',
-	default_options: ['c_std=c99', 'warning_level=3', 'libdir=lib'],
+	default_options: ['c_std=c11', 'warning_level=3', 'libdir=lib'],
         meson_version: '>= 0.51.0')
 
 cc = meson.get_compiler('c')
@@ -379,7 +379,7 @@ if get_option('static_analyze')
 if get_option('static_analyze')
 	run_target('cppcheck', command : ['cppcheck',
 									  '--quiet',
-									  '--std=c99',
+									  '--std=c11',
 									  '--suppressions-list=' + join_paths(librist_src_root, 'common/configs/cppcheck-suppressions.txt'),
 									  '--project=' + join_paths(meson.build_root(),
 									  'compile_commands.json')])
